example/main.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:http/http.dart' as http;
+import 'dart:convert' as convert;
```

Comment:
[nit] put this import first with a blank line in between

https://www.dartlang.org/guides/language/effective-dart/style#do-place-dart-imports-before-other-imports

Reply:
Done